Common/TradingCalendar.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using QuantConnect.Securities;
+using QLNet;
+
+namespace QuantConnect
+{
+    /// <summary>
+    /// Class represents trading calendar, populated with variety of events relevant to currently trading instruments
+    /// </summary>
+    public class TradingCalendar
+    {
+        private readonly MarketHoursDatabase _marketHoursDatabase;
+        private readonly SecurityManager _securityManager;
+
+        public TradingCalendar(SecurityManager securityManager, MarketHoursDatabase marketHoursDatabase)
+        {
+            _securityManager = securityManager;
+            _marketHoursDatabase = marketHoursDatabase;
+        }
+        /// <summary>
+        /// Method returns <see cref="TradingDay"/> that contains trading events associated with today's date
+        /// </summary>
+        /// <returns>Populated instance of <see cref="TradingDay"/></returns>
+        public TradingDay GetTradingDay()
+        {
+            var today = _securityManager.UtcTime.Date;
+
+            return GetTradingDay(today);
```

Comment:
Do time zones matter here? If unspecified, we would typically look at `IAlgorithm.TimeZone` as the default time zone, which could be accepted in the ctor (if it matters here)